python/feathub/examples/nyc_taxi_flink_session.py

```diff
@@ -194,8 +194,8 @@ def build_features(client: FeathubClient) -> FeatureView:
     f_trip_time_duration = Feature(
         name="f_trip_time_duration",
         dtype=types.Int32,
-        transform="unix_timestamp(CAST(lpep_dropoff_datetime AS STRING)) - "
-        "unix_timestamp(CAST(lpep_pickup_datetime AS STRING))",
+        transform="UNIX_TIMESTAMP(CAST(lpep_dropoff_datetime AS STRING)) - "
```

Comment:
Could you double-check the difference between this file and `nyc_taxi.py` and make sure that they only diff in the client configuration?

And it is possible to put the common part as method(s) in `nyc_taxi.py` and let `nyc_taxi_flink_session.py` invoke the method(s) instead of duplicating the FeatHub feature definitions?